index.js

```diff
@@ -0,0 +1,196 @@
+     "featured":"../Assets/Images/header-illsutration-mobile@2x.png", 
+     "technologies":["Ruby on rails","css","javascript","html"],
+     "live":"ldouglasot.github.io",
+     "source":"https://github.com/LDouglasOT/ldouglasot.github.io"
+  },
+
+]
+
+window.addEventListener('load', ()=>{
+
+  const untitlelist=document.querySelector('.works')
+
+  const li=document.createElement('ul')
+  li.classList.add('works-ul')
+  const insertitems=(name,description,featured,live,source,technologies)=>{
+  return `
+
+  <div class="top-notch">
+    <img class="imagedescription" src="https://wallpaperaccess.com/full/220746.jpg" alt='capstone' />
+    <div class="top-notch-content">
+      <h3 class="top-tag">
+        <span>${name}</span>
+      </h3>
+      <ul class="top-notch-content-ul">
+      
+        <li class="top-notch-content-ul-li">${technologies[0]}</li>
+        <li class="top-notch-content-ul-li">${technologies[1]}</li>
+        <li class="top-notch-content-ul-li">${technologies[2]}</li>
+        <li class="top-notch-content-ul-li">${technologies[3]}</li>
+      </ul>
+      <br />
+      <button class="top-notch-content-a" id='seeproject'>See Project</b>
+      <br />
+    </div>
+  </div>
+ 
+  `
+   }
+   projectlist.map((item)=>{
+    const lists=document.createElement('li')
+    lists.classList='list-item'
+    lists.innerHTML=insertitems(item.name, item.description, item.featured, item.live, item.source,item.technologies)
+    li.appendChild(lists)
+   })
+   untitlelist.append(li)
+});
+
+
+
+window.onload = function() {
+if (window.matchMedia("(max-width: 680px)").matches) {
+const btnclick=document.querySelector('#seeproject')
+console.log(btnclick)
+
+let pop = document.createElement('div')
+  pop.classList.add('popout')
+  pop.innerHTML=`
+  
+      <img src="../Assets/Images/SnapshootPortfolio.png" alt="Portifolio image"  class="p-image">
+    <h3 class='taghead'>Keeping track of hundreds of components</h3>
+  <h6 class='tagline'>
+    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea
+  </h6>
+ 
+  <div class="btnwrappers">
+    <a href="#"  class="see">See Live <img src='../Assets/Images/seeliveicon.png' alt="hello" /></a>
+    <a href="#" class="see">See Source <img src='../Assets/Images/Vector.png' alt="hello" /></a>
+  </div>
+  `;
+  
+  const btnsee=document.querySelector('.popupContainer')
+  btnsee.appendChild(pop)
+  btnsee.classList.remove('popupContainer')
+  btnsee.classList.add('popout-container')
+  btnsee.style.display='none'
+  const btnclose=document.querySelector('.p-image')
```

Comment:
Kindly check the layout of the `see live` and `see source` button as they are overlapping and does not stay in the pop-up container as shown below. 

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-10-14 11-26-02](https://user-images.githubusercontent.com/65233860/195837700-2f372fc5-ad97-47ca-9b34-02dab7d0a2af.png)  |  ![Screenshot from 2022-10-14 11-37-26](https://user-images.githubusercontent.com/65233860/195837766-76f87d11-dc3f-42ca-9f14-33f4f0bd5943.png)